### ISSUE TITLE
Patient name on test card should be a header

### ISSUE THREAD

**Issue by nathancrtr** [CDCgov/prime-simplereport#4054]
## Area or Component
Conduct Test

## Quick Summary
On a test card, the patient name is not marked as a heading, but functions as one (especially when there are multiple cards on the page).

Any CSS updates required?

## Accessibility Violation
* `semantic-heading`
* Text appears and functions like a section heading but is not marked up as such.

## Action Requested
* The patient name on a test card should be marked up as a header

## Source Code (optional)
https://github.com/CDCgov/prime-simplereport/blob/f11273c49f8e25f8a85449b510e6947a49d3457c/frontend/src/app/testQueue/QueueItem.tsx#L783-L793

## Original Deque Issue
https://cdc-devtools.dequecloud.com/rule-help/en/semantic-heading
